dgl_sparse/src/matrix_ops.cc

```diff
@@ -58,12 +55,112 @@ std::tuple<std::shared_ptr<COO>, torch::Tensor, torch::Tensor> COOIntersection(
   return {ret_coo, lhs_indices, rhs_indices};
 }
 
+torch::Tensor RevertIndices(const torch::Tensor& indices) {
```

Comment:
```suggestion
static torch::Tensor RevertIndices(const torch::Tensor& indices) {
```

Reply:
Use `static` to decorate internal functions in a cc file.

Reply:
Do CompactCOO and CompactCSR also need to be decorated？

Reply:
Yes.